examples/golang/actors/http-echo-tinygo/wadm.yaml

```diff
@@ -0,0 +1,44 @@
+# This is a full example of how to run the kvcounter actor exposed with an HTTP server. 
+#
+# Using this example requires you to have a Redis server running locally
+#  (though the linkdef can be modified to use a Redis server you have running elsewhere).
+#
+# You also need to have WADM running:
+# https://github.com/wasmCloud/wadm/tree/main/wadm. 
+#
+# You can deploy this example with a simple command:
+# `wash app deploy wadm.yaml`
+---
+apiVersion: core.oam.dev/v1beta1
+kind: Application
+metadata:
+  name: http-echo-tinygo-component
+  annotations:
+    version: v0.0.1
+    description: "HTTP Echo demo in Golang (TinyGo), using the WebAssembly Component Model and WebAssembly Interfaces Types (WIT)"
+spec:
+  components:
+    - name: http-echo-tinygo
+      type: actor
+      properties:
+        # TODO: you must replace the path below to match your genreated code in build
+        image: file:///home/mrman/code/work/cosmonic/forks/echo-wasi/build/http-echo-tinygo-component_s.wasm
```

Comment:
Can we make this path generic/a placeholder?

Reply:
Ah sorry this didn't make it [over during the copy](https://github.com/wasmCloud/examples/pull/245/files#diff-fe1c89408d46e810c1f20806d2a35c07e718358c887d50c256dc02d19c185330R69-R71). will fix

Reply:
Done!